pkg/streamingpromql/types/pool.go

```diff
@@ -9,16 +9,15 @@ import (
 )
 
 const (
-	maxExpectedSeriesPerResult  = 10_000_000 // Likewise, there's not too much science behind this number: this is the based on examining the largest queries seen at Grafana Labs.
-	seriesPerResultBucketFactor = 2
+	maxExpectedSeriesPerResult = 10_000_000 // Likewise, there's not too much science behind this number: this is the based on examining the largest queries seen at Grafana Labs.
```

Comment:
(nit) after previous re-arranging the "likewise" has no context here

Reply:
Added in 0552399.